src/common/helpers/api.js

```diff
@@ -27,3 +27,8 @@ export class APICompatibleError extends Error {
     this.json = { status: 'error', payload };
   }
 }
+
+// Format for Authorization header using macaroons
+export function getAuthHeader(root, discharge) {
```

Comment:
Probably clearer and less cognitive load if it were getMacaroonAuthHeader, but that's nit picking :)